modules/n1ql/pages/n1ql-language-reference/numericfun.adoc

```diff
@@ -4,102 +4,127 @@
 
 {description}
 
-ABS(expression)
+== ABS(expression)
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function